### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by NicholasBatesNZ**
README.md created

### CODE REVIEW

**Review by vampcat — CHANGES_REQUESTED** [review]
My biggest gripe is, this readme is meant for players as much as - if not more than - it is meant for developers. If I'm a person new to Terasology who just downloaded the game off the internet, saw this module called "FunnnyBlocks" and downloaded it, this readme does not really help me get started.

"Breaking Block"
But I don't see a breaking block in my inventory! Do I look around for a block, do I perform some ritual to please the block gods or do I simply type a command in the console? I don't know, and this readme does not explain that.

**Comment by NicholasBatesNZ** [comment]
@vampcat I have added a "Playing" section that I think addresses this issue

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- SpeedBoost - Launches the player in the direction of the arrow when they walk over it
+- SpeedBooster - The Speed Boost block but in a smaller, tile format
+- CheeseWheel - Just a cheese wheel, because why not?
+- SlicedCheeseWheel - A cheese wheel with a slice taken out of it. Yum!
+- BowlingPin - A tenpin bowling pin
+
+## Playing
+After installing and activating this module in a new world you have two options as to how to access the Funny Blocks:
+1. Install and activate a world generator that includes some of these blocks (see [FunnyGenerator](https://github.com/NicholasBatesNZ/FunnyGenerators) as an example). The blocks will be spawned in the world automatically.
+2. Press the \` key and enter the command `give BlockName` where "BlockName" is the name of the desired block as taken from the list above. This block will appear in your inventory.
```
Comment (mjuvekar7) [inline README.md]: change \` to `~` (even though ~ is the secondary character on the key, the key is commonly known as the tilde key)

**Comment by mjuvekar7** [comment]
The rest looks good to me :)

**Comment by Cervator** [comment]
Belated thanks for this :-)

The image hosting site changed, but I was able to figure out where the image went. But since they seemingly disallow hot linking I dumped the image file into this module's wiki repo instead, that way it doesn't bloat this repo yet is more reliably available. Both the readme and the wiki shows that image now 👍

**Pull request merged**